subprojects/samples/src/integTest/groovy/org/gradle/integtests/samples/java/SamplesJavaTestingIntegrationTest.groovy

```diff
@@ -36,6 +36,7 @@ class SamplesJavaTestingIntegrationTest extends AbstractSampleIntegrationTest {
     @UsesSample("java/basic")
     def "can execute simple Java tests with #dsl dsl"() {
         given:
+        configureExecuterForToolchains('11')
```

Comment:
Does this mean we'll lose the coverage for running these samples on higher version, like 14/15?

Reply:
No, the build itself will run with whatever JDK is configure as `testJavaHome`. However internally, it will use Java 11 for executing tasks.
Do you think that's an issue?